website/docs/zephyr-training/05_golioth/lightdb_stream.md

```diff
@@ -156,10 +156,10 @@ click on the `LightDB Stream` tab.
 
 :::tip
 
-Your device should still have Golioth credentials from earlier exercises saved
-on the settings partition that allows it to connect to the servers. You may
-check to be sure by issuing the `settings get golioth/psk-id` command in the
-serial shell.
+The "Erase & write" operation in the firmware programming steps may have wiped
+out the settings partition containing the Golioth credentials. If your device is
+not connecting to Golioth, you can check the stored PSK-ID setting by issuing
+the `settings get golioth/psk-id` command in the serial shell.
```

Comment:
When I tested this on the nRF7002 DK, the "Erase & write" operation wiped out the settings partition

Reply:
This is a good update. I have noticed this as well and suspect the behavior of the Nordic tools change... I couldn't figure out how to wipe credentials before!